app/models/fulfillment.rb

```diff
@@ -16,6 +16,7 @@ class Fulfillment
   field :check_number, type: String
   field :date_of_check, type: Date
   field :audited, type: Boolean
+  field :procedure_cost, type: Integer
```

Comment:
this field is `fund_payout` now per #1529  -- `Fulfillment#procedure_cost` was conflicting with the field with the same name on `Patient`.

Let's change the field in `seeds` to be `fund_payout`, if that's cool?

sorry I missed this, thought I got it everywhere! my mistake.